sktools/src/sktools/scripts/collectwavecoeffs.py

```diff
@@ -150,6 +151,13 @@ def setup_parser_main(parser):
         help='Logging level (default: info)')
 
 
+def get_parser_and_subparser_container():
+    parser = argparse.ArgumentParser(description=USAGE)
```

Comment:
Missing a docstring

Reply:
One could argue that this fact makes it follow the style-guide of the sktools, but you are right of course. Docstring added.